package.json

```diff
@@ -13,7 +17,7 @@
   },
   "license": "MIT",
   "engines": {
-    "node": ">= 6.0.0"
+    "node": ">= 7.6.0"
```

Comment:
Didn't you write it so that it doesn't require Node 7?

Reply:
Wops true. I'll fix.